demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -46,7 +47,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Log.setShowLogs(true);
 
-        migrationServiceBinder = MigrationFactory.migrationServiceBinder(this, migrationCallback);
+        downloadMigrationService = new MigrationService(this);
```

Comment:
No longer passing `Notification` specific classes as constructor params, these are now given through the `startMigration` method.